packages/BiostructureViewer/src/tests/molstar-preview-tests.ts

```diff
@@ -0,0 +1,51 @@
+import * as ui from "datagrok-api/ui";
+import * as grok from "datagrok-api/grok";
+import * as DG from "datagrok-api/dg";
+
+import {
+  category /*, expect*/,
+  test,
+  expect
+} from "@datagrok-libraries/utils/src/test";
+import { _package, molecule3dNglView1 } from "../package";
+import { _packageName } from "./utils";
+
+const validFileNames = ['1bdq.pdb', '1bdq.sdf', 'dc.mol2',
+  '4tkx.mmcif', 'caffeine.xyz', 'grofile.gro', 'pdbqt.pdbqt'];
+
+category("MolstarPreview", () => {
```

Comment:
please follow one style of writing strings like `'this'`

Reply:
ah, something is wrong with my linter.